src/ui/hexedit.cc

```diff
@@ -231,6 +349,13 @@ QModelIndex HexEdit::selectedChunk() {
   return chunkSelectionModel_->currentIndex();
 }
 
+void HexEdit::createAction(util::settings::shortcuts::ShortcutType type, const std::function<void ()>& f) {
+  auto action = ShortcutsModel::getShortcutsModel()->createQAction(
+        type, this, Qt::WidgetWithChildrenShortcut);
```

Comment:
This is the default indent that Qt Creator creates in such situation (in fact in previous PR there was a lot of such indents) - what indent does Google C++ style guide use in such situation?